demisto_sdk/commands/format/update_pythonfile.py

```diff
@@ -22,18 +22,19 @@ class PythonFileFormat(BaseUpdate):
     """
 
     def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '',
-                 no_validate: bool = True):
-        super().__init__(input, output, path, from_version, no_validate)
+                 no_validate: bool = True, verbose: bool = False):
+        super().__init__(input, output, path, from_version, no_validate, verbose)
 
     @staticmethod
-    def format_py_using_autopep(py_file_path):
+    def format_py_using_autopep(py_file_path, verbose: bool = False):
         """Run autopep8 formatter on python file.
         Args:
             py_file_path (str): The python file path.
         Returns:
             bool. True if succeed to run autopep8 on file, False otherwise.
         """
-        print(f"\nRunning autopep8 on file: {py_file_path}\n")
+        if verbose:
+            print(f"\nRunning autopep8 on file: {py_file_path}\n")
```

Comment:
didn't we want to shy away from the 'print` for click.secho?
https://github.com/demisto/etc/issues/23929
@yaakovi ?
relevant for all occurrences.

Reply:
wasn't aware of it.
@orlichter1, please update to `click`